doc/api/process.md

```diff
@@ -82,16 +84,15 @@ process.on('exit', (code) => {
 added: v0.5.10
 -->
 
+* `message` {Object} a parsed JSON object or primitive value.
```

Comment:
I am not sure if we should explicitly enumerate "primitive values" here. What types can they be?
`{Object | boolean | number | string | null }`?
It seems `any` does not suffice as there hardly can be `undefined` or `symbol`.

Reply:
Hm, I have no strong opinion. @Trott maybe?

Reply:
Since I did not change the current state, I guess this can be addressed at a different point of time as well?

Reply:
[Addressed)](https://github.com/nodejs/node/pull/20405)